tsconfig.json

```diff
@@ -18,8 +18,7 @@
         "noImplicitReturns": true,
         "noImplicitThis": true,
         "preserveConstEnums": true,
-        "outDir": "./lib",
-        "rootDir": "./src"
```

Comment:
Why this change? I played with. this and removing this broke the ability to use the generated package where the tests still worked.  rootDir was here to prevent that from happening.